### PR TITLE
Build for Drill 1.15.0

### PR DESCRIPTION
**Pull request by shimamoto**
https://drill.apache.org/docs/apache-drill-1-15-0-release-notes/
> Drill also supports JDK 9-11.

### CODE REVIEW

**Comment by shimamoto** [comment]
@takezoe Do you have time to review this PR?

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+jdk:
+  - oraclejdk11
+  - openjdk11
+
+services:
+  - docker
+
+before_install:
+  - mvn package -DskipTests=true
+  - docker build -t drill-test -f integration-tests/Dockerfile --build-arg java_version=11 --build-arg drill_version=1.15.0 .
```
Comment (takezoe) [inline .travis.yml]: It would be better if `drill_version` can be extracted from `pom.xml`.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+sudo: required
+
+language: java
+
+jdk:
+  - oraclejdk11
+  - openjdk11
+
+env:
+  - java_version=11 drill_version=$(grep "<drill.version>" pom.xml | cut -d ">" -f 2 | cut -d "<" -f 1)
```
Comment (shimamoto) [inline .travis.yml]: Like this?

**Comment by shimamoto** [comment]
@takezoe Thanks for reviewing!

**Pull request merged**